src/main/app/pdf/mappers/partyDetailsMapper.ts

```diff
@@ -0,0 +1,54 @@
+import { Party } from 'claims/models/details/yours/party'
+import { Company } from 'claims/models/details/yours/company'
+import { SoleTrader } from 'claims/models/details/yours/soleTrader'
+import { Organisation } from 'claims/models/details/yours/organisation'
+import { PartyType } from 'forms/models/partyType'
+
+export class PartyDetailsMapper {
```

Comment:
Not sure why that converter even exists since created structure is 99% the same. The same result is probably achievable by doing passing party instance without conversion.

CHANGE needed